CMakeLists.txt

```diff
@@ -20,6 +24,12 @@ set(ROL_ENABLE_EXAMPLES OFF CACHE BOOL "")
 set(ROL_ENABLE_TESTS OFF CACHE BOOL "")
 set(ROL_ENABLE_Sacado OFF CACHE BOOL "")
 
+# This is just making sure if MKLROOT is installed we use it
+IF(DEFINED ENV{MKLROOT})
+   SET(TPL_BLAS_LIBRARIES "$ENV{MKLROOT}/lib/intel64/libmkl_rt.so")
+   SET(TPL_LAPACK_LIBRARIES "$ENV{MKLROOT}/lib/intel64/libmkl_rt.so")
```

Comment:
I think these should be `CACHE` variables like the others, so they can be overridden if necessary.